pkg/sql/sem/tree/create.go

```diff
@@ -2147,7 +2147,7 @@ func (node *CreateExtension) Format(ctx *FmtCtx) {
 	// do not contain sensitive information and
 	// 2) we want to get telemetry on which extensions
 	// users attempt to load.
-	ctx.WriteString(node.Name)
```

Comment:
`ctx.FormatNode(node.Name)` is better right?

Reply:
`FormatNode` removes the identifier when performing redaction. Which is exactly what we don't want here.